SidebarMenu/NewsTableViewController.swift

```diff
@@ -34,7 +34,10 @@ class NewsTableViewController: UITableViewController
     
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath)
     {
-        performSegue(withIdentifier: "myPush", sender: self)
+        let alert = UIAlertController(title: "Alert", message: "Message", preferredStyle: UIAlertControllerStyle.alert)
+        alert.addAction(UIAlertAction(title: "Click", style: UIAlertActionStyle.default, handler: nil))
+        self.present(alert, animated: true, completion: nil)
+        performSegue(withIdentifier: "newsSegue", sender: self)
```

Comment:
Hi @RichardYang95, some feedback during code review:

The requirements specified asking for confirmation before navigating to the new page. This PR shows a confirmation while navigating to the new page, and does not give users the opportunity to decline navigation.

Typically, we would ensure the navigation component is in the handler of the action you added to the alert.